app-modules/assistant/src/Notifications/SendAssistantTranscriptNotification.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Assist\Assistant\Notifications;
+
+use App\Models\User;
+use Illuminate\Bus\Queueable;
+use Assist\Assistant\Models\AssistantChat;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+use Assist\Assistant\Services\AIInterface\Enums\AIChatMessageFrom;
+
+class SendAssistantTranscriptNotification extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    public function __construct(
+        protected AssistantChat $chat,
+        protected User $sender
+    ) {}
+
+    /**
+     * @return array<int, string>
+     */
+    public function via(object $notifiable): array
+    {
+        return ['mail'];
+    }
+
+    public function toMail(User $notifiable): MailMessage
+    {
+        $message = (new MailMessage())
+            ->greeting("Hello {$notifiable->name},");
+
+        if ($this->sender->is($notifiable)) {
+            $message->subject("Assistant Chat Transcript: {$this->chat->name}")
+                ->line('Here is a copy of your chat with Canyon:');
+        } else {
+            $message->subject("An Assistant Chat Transcript has been shared with you: {$this->chat->name}")
+                ->line("Here is a copy of {$this->sender->name}'s chat with Canyon:");
+        }
+
+        $this->chat
+            ->messages
+            ->each(function ($chatMessage) use ($message) {
+                if ($chatMessage->from === AIChatMessageFrom::User) {
+                    $message->line("You: {$chatMessage->message}");
```

Comment:
@mxm1070 One very small change here. Let's ensure "You" is {$sender->name} if applicable, similar to message/subject changes.